### PR TITLE
fix: patch the docs regarding the swap of the default VSX registry instance (from the public to the embedded)

### PR DESCRIPTION
**Pull request by max-cx**
## What does this pull request change?
Patch the docs regarding the swap of the default VSX registry instance (from the public to the embedded).

## What issues does this pull request fix or reference?
https://issues.redhat.com/browse/RHDEVDOCS-5396

## Specify the version of the product this pull request applies to
Merge to `main`.
Cherry-pick to `7.64.x`, `7.65.x`, `7.66.x` and `7.67.x`.

## Pull Request checklist

The author and the reviewers validate the content of this pull request with the following checklist, in addition to the [automated tests](code_review_checklist.adoc).

- Any procedure:
  - [ ] Successfully tested.
- Any page or link rename:
  - [ ] The page contains a redirection for the previous URL.
  - Propagate the URL change in:
    - [ ] Dashboard [default branding data](https://github.com/eclipse-che/che-dashboard/blob/main/packages/dashboard-frontend/src/services/bootstrap/branding.constant.ts)
    - [ ] Chectl [constants.ts](https://github.com/che-incubator/chectl/blob/main/src/constants.ts)
- [ ] Builds on [Eclipse Che hosted by Red Hat](https://workspaces.openshift.com).
- [ ] the *`Validate language on files added or modified`* step reports no vale warnings.

### CODE REVIEW

**Review by ibuziuk — APPROVED** [review]

File: modules/administration-guide/partials/proc_selecting-an-open-vsx-registry-instance.adoc [hunk 1]
```diff
@@ -3,9 +3,16 @@
 [id="selecting-an-open-vsx-registry-instance"]
 = Selecting an Open VSX registry instance
 
-The Open VSX registry at `pass:c,a,q[https://open-vsx.org]` is the default if resolved from within your organization's cluster. If not, then the embedded Open VSX registry within the {prod-short} `plugin-registry` pod is the default.
+The embedded Open VSX registry within the {prod-short} `plugin-registry` is the default.
```
Comment (svor) [inline modules/administration-guide/partials/proc_selecting-an-open-vsx-registry-instance.adoc]: the embedded Open VSX registry is default only for DevSpaces, for Che the default one is https://open-vsx.org

Comment (max-cx) [reply modules/administration-guide/partials/proc_selecting-an-open-vsx-registry-instance.adoc]: @svor and @ibuziuk, is the following still correct?
```
The Open VSX registry at `pass:c,a,q[https://open-vsx.org]` is the default if resolved from within your organization's cluster.
```
I mean does Che/DS check whether https://open-vsx.org is resolved or not? I thought it was an automated decision tree: if https://open-vsx.org is not resolved (that means Che/DS is in an airgapped environment), then Che/DS switches to using the embedded registry. And now DS does not check whether https://open-vsx.org is resolved, correct? Does Che still check whether https://open-vsx.org is resolved?

Comment (svor) [reply modules/administration-guide/partials/proc_selecting-an-open-vsx-registry-instance.adoc]: as far as I know we don't check if open-vsx.org is resolved

Comment (max-cx) [reply modules/administration-guide/partials/proc_selecting-an-open-vsx-registry-instance.adoc]: Thank you for clarifying this, @svor.

File: modules/administration-guide/partials/proc_selecting-an-open-vsx-registry-instance.adoc [hunk 2]
```diff
@@ -16,14 +20,6 @@ If the default Open VSX registry instance is not what you need, you can select a
...
-+
-[TIP]
-====
-* The default `openVSXURL` value is `pass:c,a,q[https://open-vsx.org]`.
- 
-* To select the embedded Open VSX registry instance in the `plugin-registry` pod, use `openVSXURL: ''`. You can xref:adding-or-removing-extensions-in-the-embedded-open-vsx-registry-instance[customize the list of included extensions].
-
-* You can also point `openVSXURL` at the URL of a standalone Open VSX registry instance if its URL is accessible from within your organization's cluster and not blocked by a proxy. 
-====
+<1> For example: `openVSXURL: "pass:c,a,q[https://open-vsx.org]"`.
```
Comment (max-cx) [inline modules/administration-guide/partials/proc_selecting-an-open-vsx-registry-instance.adoc:25]:
```suggestion
<1> For example: `openVSXURL: "pass:c,a,q[https://open-vsx.org]"`.
+
[TIP]
==== 

* To select the embedded Open VSX registry instance in the `plugin-registry` pod, use `openVSXURL: ''`. You can xref:adding-or-removing-extensions-in-the-embedded-open-vsx-registry-instance[customize the list of included extensions].

* You can also point `openVSXURL` at the URL of a standalone Open VSX registry instance if its URL is accessible from within your organization's cluster and not blocked by a proxy. 

====
```

**Comment by max-cx** [comment]
This PR is a patch to switch the documented default settings, and we need it for the upcoming release, hence skipping a peer language review.

**Pull request merged**